### ISSUE TITLE
Is it possible to add binary data other than images?

### ISSUE THREAD

**Issue by raid1** [Fmstrat/ownnote#191]
I'd like to attach an Audio to a note but cannot find out how.
If I am not mistaken, then please add this as a feature request.
Thanks!

**Comment by Fmstrat** [comment]
Almost a duplicate, but not. Directly related to: https://github.com/Fmstrat/ownnote/issues/31

Will attempt to implement at the same time.

**Comment by raid1** [comment]
Thanks, great news!
Sorry for not finding the other issue. I did search...

**Comment by Fmstrat** [comment]
@raid1 Not to worry, it's not really a duplicate, just a close request.

**Comment by aproposnix** [comment]
@Fmstrat So are you saying that in the future, users will be able to attach files like Evernote allows? So for example, I could attach an excel file and open it by double clicking it? 

If yes, that will be awesome! I could finally start to ditch Evernote.

**Comment by Fmstrat** [comment]
@aproposnix Yes, that is the plan. I've yet to determine if it will require a logon (once) to the ownCloud server for access to the files app, but I don't think it will. We'll see once I get into it.

**Comment by NordicTux** [comment]
Would be nice if you don't forget Video files...

**Comment by iplor** [comment]
The ability to embed audio, video, and PDF files would make this app an excellent fit for me. This would allow me to organize my research and study materials in Owncloud, which would be really nice.

When we insert a link to another file (under the current system), does Owncloud have a way of tracking each unique file, even if it is renamed or moved? I know that DMS systems like Alfresco assign each file a unique ID, so that renames and moves never cause a problem.

**Comment by traxxion** [comment]
+1

**Comment by thorstonwells** [comment]
Any progress on this issue? I was stoked to find and install this app, but was very disappointed when I couldn't attach files.

I have no need for any fancy double-click to open them, just the ability to store a snapshot of some files associated with the note and download them later if necessary would be amazing.
